core/src/main/java/edu/wpi/grip/core/GRIPCoreModule.java

```diff
@@ -48,7 +48,7 @@
                 globalLogger.removeHandler(handler);
             }
 
-            final Handler fileHandler = new FileHandler("%h/GRIP.log");//Log to the file "GRIPlogger.log"
+            final Handler fileHandler = new FileHandler("%t/GRIP.log");//Log to the file "GRIPlogger.log"
```

Comment:
Never actually finished what I was doing...  Going to move the log file into the GRIP directory.